diff --git a/MODULE.bazel b/MODULE.bazel
index 58c7ae22..ef17f8dd 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -67,7 +67,7 @@ pip.parse(
 use_repo(pip, "rules_python_publish_deps")
 
 # ===== DEV ONLY DEPS AND SETUP BELOW HERE =====
-bazel_dep(name = "stardoc", version = "0.6.2", dev_dependency = True, repo_name = "io_bazel_stardoc")
+bazel_dep(name = "stardoc", version = "0.6.2", dev_dependency = False, repo_name = "io_bazel_stardoc")
 bazel_dep(name = "rules_bazel_integration_test", version = "0.20.0", dev_dependency = True)
 bazel_dep(name = "rules_testing", version = "0.6.0", dev_dependency = True)
 
